kubernetes_deploy/production/deployment.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: laa-legal-adviser-api
+        env: prod
```

Comment:
_not a blocker_
_consistency_

`cla_public` has `env: production`

https://github.com/ministryofjustice/cla_public/blob/06666b842656b94844b605858a2ab09b4a505384/kubernetes_deploy/production/deployment.yml#L13

`fala` doesn't seem to have `env` labels.

What do you think about using `env: production` instead, for consistency across our applications? Would that cause any issues?

Reply:
Done. Also added the ENV environment variable so that the database seed doesn't happen on production